snuba/migrations/snuba_migrations/metrics/templates.py

```diff
@@ -347,8 +356,12 @@ def get_consolidated_mv_name(metric_type: str) -> str:
     return f"metrics_{metric_type}_consolidated_mv_local"
 
 
-def get_polymorphic_mv_name(metric_type: str) -> str:
-    return f"metrics_{metric_type}_polymorphic_mv_local"
+def get_polymorphic_mv_name(metric_type: str, materialization_version: int) -> str:
```

Comment:
similar to my comment above, I'd rather you make a new method called `get_versioned_polymorphic_mv_name` and leave this method definition static. I don't think you changed anything but it could be very confusing if you inadvertently did